website/blog/2022-11-30-databend-weekly-70.md

```diff
@@ -0,0 +1,185 @@
+---
+title: 'This week in Databend #70'
+date: 2022-11-30
+slug: 2022-11-30-databend-weekly
+tags: [databend, weekly]
+authors:
+- name: PsiACE
+  url: https://github.com/psiace
+  image_url: https://github.com/psiace.png
+---
+
+[Databend](https://github.com/datafuselabs/databend) is a powerful cloud data warehouse. Built for elasticity and efficiency. Free and open. Also available in the cloud: <https://app.databend.com> .
+
+## What's Changed
+
+Below is a list of some major changes that we don't want you to miss.
+
+### Exciting New Features :sparkles:
+
+**format**
```

Comment:
Format

Reply:
Please capitalize all words at the same level as this one. For example, query -> Query.